Sources/EduIDExpansion/Flows/Home/MainCoordinator.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+class MainCoordinator: CoordinatorType, HomeNavigationDelegate, OnBoardingMainNavigationDelegate, ScanMainNavigationDelegate, SecurityMainNavigationDelegate, PersonalInfoMainNavigationDelegate {
```

Comment:
Tip: put each conformance into its own extension. That helps with reading the code and knowing for which conformance a method is needed.

```swift
extension MainCoordinator: HomeNavigationDelegate {
   ...
}

extension MainCoordinator: OnBoardingMainNavigationDelegate {
    ...
}
```

Reply:
I did this